kafka/source/cmd/controller/main.go

```diff
@@ -17,56 +17,14 @@ limitations under the License.
 package main
 
 import (
-	"log"
+	"knative.dev/eventing-contrib/kafka/source/pkg/reconciler"
+	"knative.dev/pkg/injection/sharedmain"
+)
 
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-	"knative.dev/eventing-contrib/kafka/source/pkg/apis"
-	controller "knative.dev/eventing-contrib/kafka/source/pkg/reconciler"
-	"knative.dev/pkg/logging/logkey"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+const (
+	component = "kafka-controller"
```

Comment:
can we do `kafka_controller` here? 

https://github.com/knative/eventing/pull/1747/files